lib/trento/application/usecases/sap_systems/health_summary_service.ex

```diff
@@ -60,16 +64,24 @@ defmodule Trento.SapSystems.HealthSummaryService do
     |> HealthService.compute_aggregated_health()
   end
 
-  @spec compute_clusters_health(instance_list) :: Health.t()
-  defp compute_clusters_health(instances) do
+  @spec compute_cluster_health(instance_list, [ClusterType.t()]) :: Health.t()
+  defp compute_cluster_health(instances, cluster_types) do
     instances
     |> reject_unclustered_instances()
     |> clusters_from_instance()
-    |> keep_only_hana_scale_up_clusters()
+    |> filter_by_cluster_type(cluster_types)
     |> health_from_cluster()
     |> HealthService.compute_aggregated_health()
   end
 
+  @spec reject_unclustered_instances(instance_list) :: instance_list
```

Comment:
Just changed some functions order...
They were not following the order of appearance